yautpa-desktop/src/index.scss

```diff
@@ -1,7 +1,10 @@
 @import "style/vars.scss";
 
 body {
-    background-color: rgb(11, 5, 48);
+    background: rgb(0,117,0);
```

Comment:
Is this just a backup for browsers that dont support linear gradient or is this unnecesary? Because electron uses chromium, which has always supported this

Reply:
Seems like it. tbh I copied it from a website

Reply:
As this does not affect anything, it can stay in the code. But while you're working on the requested changes, you can just put a comment explaining why that line is there